packages/testkit/__TESTS__/textfield.spec.js

```diff
@@ -0,0 +1,20 @@
+import { test, expect } from "@playwright/test";
+import { TextField } from "../inputs/TextField";
+
+test.describe("textArea Class with Storybook", () => {
+  let textField;
+  let textfieldLocator
+
+  test.beforeEach(async ({ page }) => {
+    await page.goto("/?path=/story/inputs-textfield--overview");
```

Comment:
Is there a better way to handle it? If the paths will change it will break tests

Reply:
i moved the paths to a separate file it would make it easier to maintain that way but I don't see anyway we could avoid breaking tests if paths change because it purely depends on core.